lib/mysql.js

```diff
@@ -272,6 +272,11 @@ MySQL.prototype.toFields = function (model, data) {
       if (undefined === value) {
         return;
       }
+      // skip property if it cannot be NULL, but `toDatabase` function returns NULL
+      if(props[key].mysql && props[key].mysql.nullable &&
```

Comment:
@raymondfeng do you mean that `nullable` can be set to `true` or `false`, not only `"Y"` or `"N"`